src/main/java/com/leetcodeprep/linkedList/LinkedListCustom.java

```diff
@@ -0,0 +1,77 @@
+package com.leetcodeprep.linkedList;
+
+public class LinkedListCustom <V>{
+    private LinkedListNode head;
+    private LinkedListNode tail;
+    private int nodeCount;
+
+    public LinkedListNode add(V val){
+        if(head == null){
+            head = new LinkedListNode();
+            head.value = val;
+            tail = head;
```

Comment:
To be honest I don't understand why are the tail and the head are the same. What is the purpose to have tail in this case? 

In generic algorithm tail and head are different links. And when it is, you can make some optimization on your code, for example, in the method get(). (think how to optimize get in that case)
![image](https://github.com/ImranArif536/LeetcodePrep/assets/2914310/5cf6c9d4-f5f7-437d-9047-85f3460d8436)